### ISSUE TITLE
futureVersionTag

### ISSUE THREAD

**Issue by peterfigure** [joffrey-bion/gradle-github-changelog#44]
Hi - making sure i'm understanding this feature correctly, when generating the changelog I want to generate it to include the version I'm about to tag (but I'm deferring tagging until after the changelog is generated so to include it in the tag). 

I know the version of the tag and set `futureVersionTag` to this value but it only generates the changelog until the previously tagged version. Is there a way to do what I want?

**Comment by joffrey-bion** [comment]
Hi, yes this is exactly what `futureVersionTag` is meant for. Could you please share the rest of your config?

Make sure `showUnreleased` is true (which is the default), otherwise the unreleased issues (after the last tag) won't be part of the changelog

**Comment by peterfigure** [comment]
hard to convey an example succinctly, but here we go:

current state:
```
at 13:59:03 ❯ git log --oneline --decorate | head -10
b22264f (HEAD -> main, tag: v0.0.23, origin/main) always print out the calculated version, add changelog (#18)
2c34b31 Update README.md
54466f8 update README
7fe0986 (tag: v0.0.22) add GitHub Actions pull_request event support (#17)
```

commit some changes on main:
```
gradle-semver-plugin on  main [!] via ☕ v17.30 via 🅺 on ☁️  (us-west-2) on ☁️ pvanrensburg(us-east1) 
at 13:59:22 ❯ git add build.gradle.kts gradle/libs.versions.toml 

gradle-semver-plugin on  main [+] via ☕ v17.30 via 🅺 on ☁️  (us-west-2) on ☁️ pvanrensburg(us-east1) 
at 13:59:28 ❯ git commit -m "add benmanes versions plugin"
[main a86f711] add benmanes versions plugin
 2 files changed, 4 insertions(+), 1 deletion(-)
```

generate changelog:
```
at 13:59:43 ❯ ./gradlew cv generateChangelog
Type-safe dependency accessors is an incubating feature.

> Configure project :
semver: 0.0.24
```

changelog contents:
```
at 14:00:10 ❯ head -10 CHANGELOG.md 
# Change Log

## [v0.0.23](https://github.com/nefilim/gradle-semver-plugin/tree/v0.0.23) (2022-01-10)
[View commits](https://github.com/nefilim/gradle-semver-plugin/compare/v0.0.22...v0.0.23)

**Merged pull requests:**

- always print out the calculated version, add changelog [\#18](https://github.com/nefilim/gradle-semver-plugin/pull/18) ([@nefilim](https://github.com/nefilim))

## [v0.0.22](https://github.com/nefilim/gradle-semver-plugin/tree/v0.0.22) (2022-01-10)
```

changelog configuration here: https://github.com/nefilim/gradle-semver-plugin/blob/main/build.gradle.kts#L142

**Comment by peterfigure** [comment]
I expect the changelog to contain [v0.0.24] with the commit I've just added

**Comment by joffrey-bion** [comment]
This changelog generator lists closed issues and merge pull requests, not individual commits (at least not directly, only in the "view commits" link).

The idea behind this is that the changelog is for the consumers of the library/application, which are interested in more high-level issues, while commits are for developers.

If there are no issues closed after the tag `v0.0.23`, it is expected that nothing is visible in the changelog after that tag.

**Comment by peterfigure** [comment]
ok thank you!
